js/ui/calendar/ui.calendar.js

```diff
@@ -687,7 +729,13 @@ const Calendar = Editor.inherit({
         const $view = $('<div>').appendTo(this._$viewsWrapper);
         const config = this._viewConfig(date);
 
-        return this._createComponent($view, specificView, config);
+        const view = this._createComponent($view, specificView, config);
+
+        if(this.option('views') !== 1) {
+            $view.addClass(CALENDAR_MULTIVIEW_CLASS);
```

Comment:
May be better will be use `toggleClass(className, state)` method?

Reply:
fair point, it allows to get rid of if, done